userspace/libpman/CMakeLists.txt

```diff
@@ -35,23 +36,15 @@ target_include_directories(
 			$<BUILD_INTERFACE:${LIBS_DIR}> # ppm_enum and tables
 			$<BUILD_INTERFACE:${LIBS_DIR}/userspace> # scap-stats struct
 			${ZLIB_INCLUDE}
-			${LIBBPF_INCLUDE}
 			${MODERN_BPF_SKEL_DIR}
-			${LIBELF_INCLUDE}
```

Comment:
@FedeDP : you see how we no longer need to tell libpman to go look for libelf? libpman does not use libelf directly. This is possible because we are building libelf as a native cmake target